arcade/arcade.go

```diff
@@ -33,12 +36,21 @@ var CurrentMatches = Arcade{
 	Counter: 0,
 }
 
+func GetOpponent(player *player.Player) *player.Player {
+	match := CurrentMatches.List[player.MatchId]
+	if player.Id == 1 {
+		return match.P2
+	} else {
+		return match.P1
+	}
+}
+
 // Creates a new match, stores it in Arcade map, returns it's address
 func NewMatch(aiP1 bool, aiP2 bool) *Match {
```

Comment:
Should there be an initial first suggestion? (e.g. center X, center Y)

Reply:
yep, cool, I'll add it ;)